components/table/__tests__/Table.pagination.test.js

```diff
@@ -79,14 +79,24 @@ describe('Table.pagination', () => {
     expect(wrapper.find('.ant-pagination.mini')).toHaveLength(1);
   });
 
-  // TODO
   it('should scroll to first row when page change', () => {
     const wrapper = mount(createTable({ scroll: { y: 20 } }));
+    const scrollToSpy = jest.spyOn(
+      wrapper
+        .find('Table')
+        .first()
+        .instance(),
+      'scrollToFirstRow',
+    );
 
     wrapper
       .find('Pager')
       .last()
       .simulate('click');
+    expect(scrollToSpy).toHaveBeenCalled();
+
+    wrapper.setProps({ pagination: { pageSize: 2 } });
+    expect(scrollToSpy).toHaveBeenCalled();
```

Comment:
should be use `toHaveBeenCalledTimes`

Reply:
Thank you, I will modify it as soon as possible.